superset-frontend/spec/javascripts/sqllab/ResultSet_spec.jsx

```diff
@@ -38,49 +39,48 @@ import {
   stoppedQuery,
   initialState,
   user,
+  queryWithNoQueryLimit,
 } from './fixtures';
 
 const mockStore = configureStore([thunk]);
 const store = mockStore(initialState);
-
-describe('ResultSet', () => {
-  const clearQuerySpy = sinon.spy();
-  const fetchQuerySpy = sinon.spy();
-  const reRunQuerySpy = sinon.spy();
-  const mockedProps = {
-    actions: {
-      clearQueryResults: clearQuerySpy,
-      fetchQueryResults: fetchQuerySpy,
-      reRunQuery: reRunQuerySpy,
-    },
-    cache: true,
-    query: queries[0],
-    height: 140,
-    database: { allows_virtual_table_explore: true },
-    user,
-    defaultQueryLimit: 1000,
-  };
-  const stoppedQueryProps = { ...mockedProps, query: stoppedQuery };
-  const runningQueryProps = { ...mockedProps, query: runningQuery };
-  const cachedQueryProps = { ...mockedProps, query: cachedQuery };
-  const failedQueryWithErrorMessageProps = {
-    ...mockedProps,
-    query: failedQueryWithErrorMessage,
-  };
-  const failedQueryWithErrorsProps = {
-    ...mockedProps,
-    query: failedQueryWithErrors,
-  };
-  const newProps = {
-    query: {
-      cached: false,
-      resultsKey: 'new key',
-      results: {
-        data: [{ a: 1 }],
-      },
+const clearQuerySpy = sinon.spy();
+const fetchQuerySpy = sinon.spy();
+const reRunQuerySpy = sinon.spy();
+const mockedProps = {
+  actions: {
+    clearQueryResults: clearQuerySpy,
+    fetchQueryResults: fetchQuerySpy,
+    reRunQuery: reRunQuerySpy,
+  },
+  cache: true,
+  query: queries[0],
+  height: 140,
+  database: { allows_virtual_table_explore: true },
+  user,
+  defaultQueryLimit: 1000,
+};
+const stoppedQueryProps = { ...mockedProps, query: stoppedQuery };
+const runningQueryProps = { ...mockedProps, query: runningQuery };
+const cachedQueryProps = { ...mockedProps, query: cachedQuery };
+const failedQueryWithErrorMessageProps = {
+  ...mockedProps,
+  query: failedQueryWithErrorMessage,
+};
+const failedQueryWithErrorsProps = {
+  ...mockedProps,
+  query: failedQueryWithErrors,
+};
+const newProps = {
+  query: {
+    cached: false,
+    resultsKey: 'new key',
+    results: {
+      data: [{ a: 1 }],
     },
-  };
-
+  },
+};
+describe('ResultSet', () => {
```

Comment:
Let's go straight to the test without nesting in describe. That's the approach we have been trying to use lately